.enb/lib/page.js

```diff
@@ -0,0 +1,149 @@
+var techs = require('./techs');
+
+module.exports = function(node, options) {
+    var levels = options.levels || [],
+        browsers = options.browser || [],
+        langs = options.langs || [],
+        templateEngine = options.templateEngine || 'BH';
+
+    // essential
+    node.addTechs([
+        [techs.bem.levels, { levels : levels }],
+        [techs.bem.bemjsonToBemdecl, {
+            target : '.tmp.bemdecl.js'
+        }],
+        [techs.bem.deps, {
+            bemdeclFile : '.tmp.bemdecl.js',
+            target : '.tmp.deps.js'
+        }],
+        [techs.bem.files, {
+            depsFile : '.tmp.deps.js'
+        }]
+    ]);
+
+    // css
+    node.addTech([techs.css, {
+        target : '.tmp.css',
+        browsers : browsers
+    }]);
+    node.mode('development', function() {
+        node.addTech([techs.files.copy, {
+            source : '.tmp.css',
+            target : '?.css'
+        }]);
+    });
+    node.mode('production', function() {
+        node.addTech([techs.process, {
+            source : '.tmp.css',
+            target : '?.css',
+            minify : true,
+            tech : 'cleancss'
+        }]);
+    });
+    node.addTarget('?.css');
+
+    // JavaScript for browsers
+    node.addTechs([
+        // Template engine for JavaScript
+        [techs.bem.depsByTechToBemdecl,  {
+            target : '.tmp.js-template.bemdecl.js',
+            sourceTech : 'js',
+            destTech : 'bemhtml'
+        }],
+        [techs.bem.deps, {
+            target : '.tmp.js-template.deps.js',
+            bemdeclFile : '.tmp.js-template.bemdecl.js'
+        }],
+        [techs.bem.files, {
+            depsFile : '.tmp.js-template.deps.js',
+            filesTarget : '.tmp.js-template.files',
+            dirsTarget : '.tmp.js-template.dirs'
+        }],
+        templateEngine === 'BH'? [techs.engines.bhClient, {
+            target : '.tmp.js-template.js',
+            filesTarget : '.tmp.js-template.files',
+            jsAttrName : 'data-bem',
+            jsAttrScheme : 'json',
+            mimic : 'BEMHTML'
+        }] : [techs.engines.bemhtml, {
+            target : '.tmp.js-template.js',
+            filesTarget : '.tmp.js-template.files',
+            devMode : false
+        }],
+
+        // JavaScript Files
+        [techs.bem.depsByTechToBemdecl, {
+            target : '.tmp.required-js.bemdecl.js',
+            sourceTech : 'js',
+            destTech : 'js'
+        }],
+        [techs.bem.mergeBemdecl, {
+            sources : ['.tmp.bemdecl.js', '.tmp.required-js.bemdecl.js'],
+            target : '.tmp.js.bemdecl.js'
+        }],
```

Comment:
I reopened old [issue](https://github.com/enb-bem/enb-bem-techs/issues/76) in this topic. I propose to continue the discussion there.